### DISCUSSION TITLE
Training the model done now what

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#657] [General Q&A]
I trained my model and now speak in Spanish my language but sounds robotic ,how i can make it talk  like the dataset ?
I need to train now the vocoder or what ?

**Comment by thorstenMueller** [comment]
Quality always depends on lots of aspects (eg. quality of recordings, number of recordings/duration, number of model training steps, ...). But training a vocoder is a good idea. Depending on your use case you can choose between several available vocoders. They differ on RTF and quality.

**Reply by tiomaldy** [reply]
But i don t know something ,the vocoder train with the same dataset or how you bind your trained model with the vocoder.
I suppose that you first train the syntethizer and the you train the vocoder but i think that you train the vocoder with the model that you trained in the syntethizer.
